forem/forem
# Remove GDPR from the "users" namespace

## What type of PR is this? (check all applicable)


## Description

Remove the `users` namespace for GDPR Delete Requests. 

**Context:**
In an upcoming PR we will be placing the gdpr_delete_requests and invitations route under the member manage scope. 

Prior to this PR, the urls looked as follows: 
- Invitations: http://localhost:3000/admin/invitations
- GDPR: http://localhost:3000/admin/users/gdpr_delete_requests

Now you can see that there is a bit of inconsistency above with one of these routes being namespaced under Users and the other not. My initial reaction was to namespace the  invitations under Users. However on further reflection I've decided to remove the User namespace on GDPR instead for the following reasons: 

1. Previously these tabs lived on the Users page, hence a Users namespace made more sense. In an upcoming PR that will be moved to the sidebar, hence perhaps they don't need to be so closely correlated anymore. 
2. In the sidebar we will be grouping these routes under a scope called `:member_manager`, hence if we kept the users namespace the URL would have read `http://localhost:3000/admin/member_manager/users/gdpr_delete_requests`, with `member_manager` in the URL `/users` seems redundant. 

<img width="219" alt="Screenshot 2022-05-02 at 09 30 20" src="https://user-images.githubusercontent.com/2786819/166199725-4f4eaad0-454b-4fc4-8f17-dc2803c2fb26.png">

## Related Tickets & Documents



- Related Issue #https://github.com/forem/forem/issues/17454
- Closes #https://github.com/forem/forem/issues/17454

## QA Instructions, Screenshots, Recordings

1. I've tested and checked for missed `gdpr_delete_requests` namespace references, but a second set of eyes wouldn't harm anyone :)
2. Play around on with the page, delete a user and then delete the GDPR and ensure it all still works with no errors. 

### UI accessibility concerns?

None, moved some code around.

## Added/updated tests?

      have not been included_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/f8tRKuBIffxSpYHAwK/giphy-downsized-large.gif)
